msteams-platform/graph-api/activity-feed/feed-notifications.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Activity Feed notifications in Teams
+description: Describes activity feed notifications in Teams and how to use to the best advantage
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams notifications feed activity Graph
+---
+# Microsoft Teams activity feed notifications  — Developer Preview
+
+>[!NOTE]
+>Microsoft Teams Graph API activity feed notifications are currently available in desktop and Android clients after Developer Preview has been enabled. See [How do I enable Developer Preview](../../resources/dev-preview/developer-preview-intro.md) for more information.
+
+The Activity feed API is a Microsoft Teams and Microsoft Graph integration that enables your app to use API endpoints to notify  users of events ranging from tasks requiring a user's attention, to actions taken by others that affect the user or a set of users.
+
+## Activity feed senders and recipients
```

Comment:
feels like an abrupt start -- shouldn't there be additional intro material before we jump into this? ( And perhaps the answer is "yes there should be eventually, but you asked for a min-bar doc fast" :) )